test/test.js

```diff
@@ -12,7 +12,8 @@ test('passing files and their JSON equivalents', _=> {
         if (extension === "jams") {
 	    test(filename, t=> {
 		const jamspath = `${subpath}/${filename}`
-	        const jams_o = jams(readFileSync(jamspath))
+            var read = readFileSync(jamspath, {encoding: 'utf8'})
+	        const jams_o = jams(read)
 	        const jsonfile = readFileSync(jamspath.replace('jams', 'json'))
 	        const json_o = JSON.parse(jsonfile)
 	        t.deepEqual(Object.keys(jams_o), Object.keys(json_o))
```

Comment:
This is for line - `t.ok(jams_o[key])`.

Since `t.ok` checks for truthy value:

```
  ok (actual, msg) {
    if (this.strict && !msg) throw new Error('tapzero msg required')
    this._assert(
      !!actual, actual, 'truthy value',
      msg || 'should be truthy', 'ok'
    )
  }
```

To check if the key exists in object I think it should be this: 

`t.ok(jams_o[key] !== undefined)`

Reply:
this change is not needed. it's checking for a truthy value not a truth value

Reply:
So this one, 1 of the testcases is failing (I should've mentioned this). And it's the empty quotes.

JAMS
```
{
  key-one ""
}
```

And "" is falsey. 

But you guys are right - line 18 already checks for equality for the keys `t.deepEqual(Object.keys(jams_o), Object.keys(json_o))`, so there is no need for this.